services/migrations/20180913193732-create-tainted.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('tainteds', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      uuid: {
+        type: Sequelize.UUID
+      },
+      updateId: {
+        type: Sequelize.INTEGER,
+        references: {
+          model: 'updates',
+          key: 'id'
+        },
+      },
```

Comment:
Not a big deal anyway if there's dupes, but I guess we should define a unique composite key. IIUC, this would just be adding like
`unique: 'instance-tainted'` on both `uuid` and `updateId` fields.

Reply:
good suggestion, will add